BuildingSync.xsd

```diff
@@ -15966,6 +15931,14 @@
       </xs:simpleContent>
     </xs:complexType>
   </xs:element>
+  <xs:element name="HeatingPlantID">
```

Comment:
It was defined locally only under `SpaceHeatingSystem` before. Now it is referenced under `HeatingSource`, `HeatPump` and `SpaceHeatingSystem`, so I made it globally defined here so it can be referenced multiple times to avoid duplicated elements.

Reply:
Oh duh, I see! This is the new element we're referencing (created to deduplicate). Thanks!